.travis.yml

```diff
@@ -22,6 +22,7 @@ rvm:
   - 2.0
   - 2.1
   - 2.2
+  - 2.3.1
```

Comment:
Unfortunately, Travis hasn’t defined an `rvm alias` for `2.3`, so specifying the full version is necessary.